src/components/footer/footer.js

```diff
@@ -1,17 +1,83 @@
 import React from 'react';
+import preval from 'preval.macro';
+import moment from 'moment';
+
+const buildTimestamp = preval`module.exports = new Date().getTime();`;
 
 const Footer = () => {
 	return (
-        <footer className="w-full bg-gray-800 text-gray-300 text-xs">
-            © Copyright SSW 1990-2019. All Rights Reserved.
-            <hr/>
-            Our website is under <a href="https://rules.ssw.com.au/WebSites/RulestoBetterWebsites-Deployment/Pages/Do-your-developers-deploy-manually.aspx">CONSTANT CONTINUOUS DEPLOYMENT</a>. 
-            The lastest deployment is xxx
-        </footer>
-    );
-}
-
-Footer.propTypes = {
+		<footer className="w-full bg-black">
+			<div className="py-2 flex">
+				<div className="w-1/6" />
+				<div className="w-1/6">
+					Copyright © SSW 1990 - {new Date().getFullYear()}. All Rights
+					Reserved.
+				</div>
+				<div className="w-3/6 text-right">
+					<a className="text-gray-800" href="mailto:info@ssw.com.au">
+						FEEDBACK TO SSW
+					</a>
+					&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;
```

Comment:
Could you please use a spacer class with paddings instead ?
Make sure you use "em" sizes so it scales with zooms as well

Reply:
Done